extensions/blocks/podcast-player/templates/podcast-header-title.php

```diff
@@ -27,6 +28,7 @@
 		<span
 			class="jetpack-podcast-player__current-track-title <?php echo esc_attr( $primary_colors['class'] ); ?>"
 			<?php echo isset( $primary_colors['style'] ) ? 'style="' . esc_attr( $primary_colors['style'] ) . '"' : ''; ?>
+			<?php echo $is_amp ? '[text]="podcastPlayer.tracks[currentTrack].title"' : ''; ?>
```

Comment:
Again I think this AMP exception is fine.

I'm starting to think a good flow would be to add some documentation around usage of AMP and when to extract vs when to leave in file and the approaches we are taking. Could we have a Block-specific `README`?